File: app/src/main/java/com/urt1rs/donkeywiki/DonkeyWikiApplication.java
package com.urt1rs.donkeywiki;

import android.app.Application;
import android.content.Context;
import android.os.Bundle;

public class DonkeyWikiApplication extends Application {

    private static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = getApplicationContext();
    }

    public static Context getAppContext() {
        return mContext;
    }

}

File: app/src/main/java/com/urt1rs/donkeywiki/DonkeyWikiApplication.kt
package com.urt1rs.donkeywiki

import android.app.Application
import android.content.Context
import android.os.Bundle
import com.urt1rs.donkeywiki.helper.PrefHelper

class DonkeyWikiApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        appContext = applicationContext
    }

    companion object {
        lateinit var appContext: Context
    }

}
